commands/pfadd.md

```diff
@@ -2,7 +2,7 @@ Adds all the element arguments to the HyperLogLog data structure stored at the v
 
 As a side effect of this command the HyperLogLog internals may be updated to reflect a different estimation of the number of unique items added so far (the cardinality of the set).
 
-If the approximated cardinality estimated by the HyperLogLog changed after executing the command, `PFADD` returns 1, otherwise 0 is returned. The command automatically creates an empty HyperLogLog structure (that is, a Redis String of a specified length and with a given encoding) if the specified key does not exist.
+If the approximated cardinality estimated by the HyperLogLog changed after executing the command, `PFADD` returns 1, otherwise 0 is returned. The command automatically creates an empty HyperLogLog structure (that is, a Valkey String of a specified length and with a given encoding) if the specified key does not exist.
```

Comment:
```suggestion
If the approximated cardinality estimated by the HyperLogLog changed after executing the command, `PFADD` returns 1, otherwise 0 is returned. The command automatically creates an empty HyperLogLog structure (that is, a String of a specified length and with a given encoding) if the specified key does not exist.
```
I'm going to stop commenting on this, I guess either you'll /s/Valkey String/string or not.